components/core/blocks/Form/BaseTextarea.vue

```diff
@@ -22,19 +22,12 @@
         {{ placeholder }}
       </label>
     </div>
-
-    <ValidationMessages v-if="validations" :validations="validations" />
```

Comment:
`validations` prop was not provided in ThankYou page (the only place where `BaseTextarea` is used) and this page will be re-witten soon anyway.